src/lab8.py

```diff
@@ -369,7 +360,6 @@ def __init__(self):
         self.tabs = []
         self.active_tab = None
         self.focus = None
-        self.address_bar = ""
```

Comment:
why remove this?

Reply:
I think it now lives in the Chrome

Reply:
I guess this was just a bug?

Reply:
Sure… more like a harmless leftover from a refactor.